governance/votes/Executive vote - September 13, 2023.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Template - [Executive Vote] Stability Scope Parameter Changes, Spark Protocol D3M Parameter Changes, Set Fortunafi Debt Ceiling to Zero DAI, DAO Resolution for HV Bank, Delegate Compensation and Other Actions - September 13, 2023
+summary: Stability Scope parameter changes; Spark Protocol D3M parameter changes; set Fortunafi (RWA005-A) Debt Ceiling to zero DAI; approve DAO Resolution for HV Bank (RWA009-A) to terminate future purchases; Aligned Delegate compensation for August 2023; Core Unit MKR vesting transfers; scuttle MCD_CAT contract; trigger Spark Protocol Proxy Spell.
+date: 2023-09-13T00:00:00.000Z
+address: "$spell_address"
+
+---
+
+# [Executive Proposal] Stability Scope Parameter Changes, Spark Protocol D3M Parameter Changes, Set Fortunafi Debt Ceiling to Zero DAI, DAO Resolution for HV Bank, Delegate Compensation and Other Actions - September 13, 2023
+
+The Reserve Governance Facilitators, Sidestream, Dewiz and Phoenix Labs have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following actions within the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following actions will occur within the Maker Protocol:
+
+- Several Stability Scope parameter changes will be executed, including changes to Stability Fees and activation of the DC-IAM for PSM-PAX-A. 
+- Adjustments will be made to the Spark Protocol Dai Direct Deposit Module (D3M) Maximum Debt Ceiling and Ceiling Increase Cooldown. 
+- The Fortunafi vault's Debt Ceiling will be set to zero DAI, per the Stability Scope Artifact's requirement to offboard legacy Legal Recourse Assets.
+- A DAO resolution pertaining to the HV Bank vault will be approved to terminate future purchases and return available cash from the HVB Master Participation Trust. This is per the Stability Scope Artifact's requirement to offboard legacy Legal Recourse Assets.
```

Comment:
I would suggest removing the second sentence. It is not part of the executive summary of actions.